IES_Engine/Convert/Environment/Panel.cs

```diff
@@ -53,14 +53,25 @@ public static List<string> ToIESShading(this List<Panel> panelsAsShade, Settings
 
             for (int x = 0; x < panels.Count; x++)
             {
+                bool isTranslucent = panels[x].Type != PanelType.Shade;
```

Comment:
Put this before the for loop and set it as `panels.First().Type` instead of `panels[x].Type`